src/generator/AutoRest.Go/rakefile.rb

```diff
@@ -13,18 +16,18 @@
 	compute: {version: "2016-03-30"},
 	# containerservice: {version: "2016-03-30", swagger: "containerservice"},
 	# commerce: {version: "2015-06-01-preview"},
-	# datalake_analytics: {
-	# 	account: {version: "2015-10-01-preview"},
-	# 	catalog: {version: "2015-10-01-preview"},
-	# 	job:  {version: "2016-03-20-preview"}
-	# },
+	datalake_analytics: {
+		account: {version: "2015-10-01-preview"},
+		# catalog: {version: "2016-06-01-preview"},
```

Comment:
As per my talk with Ben Goldsmith, They do not wanted to include Data Plane services into ARM namespace intentionally but they did for .net and nodeJS. Now they cannot remove it with reasons on break their existing customer and for backward compatibilities. 

Go language still have a chance to avoid this case by not including them in the ARM namespace. I'd still prefer not to include those in the ARM namespace today or we'll be stuck in the same situation. We should only generate management planes for Store and Analytics services. What are your views on this?

---
In reply to: [79500366](https://github.com/Azure/autorest/pull/1411#discussion_r79500366) [](ancestors = 79500366)